source/Classroom/Basics/Billing/how_to_read_your_invoice.md

```diff
@@ -1,86 +1,69 @@
 
- 
+![]({{root_url}}/images/invoice_legend.png)
 
-# 1. Physical Address
+{% anchor h2 %}
+1. Physical Address
+{% endanchor %}
 
 **The physical address of your business.** This address is the same address you entered when you first signed up. If you would like to update the address as it appears on your invoices, you can click on the "Change Invoice Address" button on the [Plan & Billing Details](https://app.sendgrid.com/settings/billing) page. If you need to change the address on an existing invoice, please send an email to [billing@sendgrid.com](mailto:billing@sendgrid.com).
 
  
 
-# 2. Billing Email
+{% anchor h2 %}
+2. Billing Email
+{% endanchor %}
 
-**The email address that your invoices are sent to.** This address defaults as the primary email address on file for your account, but can be updated to a different address from the [Plan & Billing Details](https://app.sendgrid.com/settings/billing) page. If you override the billing email address in this way, the primary email address on file will stop receiving invoices. 
+**The email address where you receive SendGrid statements.** By default, this is the primary email address on file in your account, but you can update it to a different address from the [Plan & Billing Details](https://app.sendgrid.com/settings/billing) page. If you override the billing email address in this way, the primary email address on file will stop receiving invoices. 
 
 Need to send invoices to multiple addresses? Consider creating a [role address](https://sendgrid.com/docs/Classroom/Deliver/Address_Lists/role_addresses.html) on your side that includes the all of billing stakeholders in your organization, or send an email to [billing@sendgrid.com](mailto:billing@sendgrid.com).
 
- 
-
-# 3. Charge Summary
+{% anchor h2 %}
+3. Charge Summary
+{% endanchor %}
 
-This section displays the total charges for services used **in the current and/or previous month.** 
+These are the total charges for services used **in the current and previous month**: 
 
 ![]({{root_url}}/images/invoice_NL_usage.png)
 
-**Newsletter Usage** - This item refers to the total usage charge for sending during the previous month with Legacy Newsletter features via either the API or the UI.
-
-Please refer to "Newsletter Usage" in the Usage Summary section below for more information on how this charge is calculated. 
-
- 
-![]({{root_url}}/images/invoice_pro100k_1.png)
-
-*This item refers to any charge for the previous month for the base plan price. Under normal circumstances, this item will always show a charge of $0.00 for this service period.
-
+This describes which plan you are on and describes the total usage charge for sending during the previous month. In this case, the user switched from the Essentials plan to the Pro plan in the middle of the previous month, and the charge reflects a credit for the part of the month they weren't on Essentials and a charge for the part of the month where they switched to Pro.
```

Comment:
This describes the plan you subscribe to and the total charge for usage in the previous month.